rich/control.py

```diff
@@ -182,6 +198,22 @@ def strip_control_codes(
     return text.translate(_translate_table)
 
 
+def make_control_codes_readable(
```

Comment:
I'd prefer `escape_control_code`. Would you mind changing the terminology from "make readable" elsewhere?